### PR TITLE
Add logger and add function to update config params of all modules fr…

### PR DESCRIPTION
**Pull request by javibu13**
…om one file

### CODE REVIEW

File: code/sys_conf/src/system_config_tool/sys_conf.py [hunk 1]
```diff
@@ -1,36 +1,38 @@
...
+'''
+This module works to have a configuration, mainly for the logger.
+'''
 #######################        MANDATORY IMPORTS         #######################
 from __future__ import annotations
-import sys
-import os
 
 #######################         GENERIC IMPORTS          #######################
+import sys
```
Comment (mariuscrsn) [inline code/sys_conf/src/system_config_tool/sys_conf.py]: Import only path

File: code/sys_conf/src/system_config_tool/sys_conf.py [hunk 2]
```diff
@@ -1,36 +1,38 @@
...
+from os import getcwd, getenv
 
 #######################       THIRD PARTY IMPORTS        #######################
 import yaml
 
-sys.path.append(os.getcwd())  #get absolute path
+sys.path.append(getcwd())  #get absolute path
```
Comment (mariuscrsn) [inline code/sys_conf/src/system_config_tool/sys_conf.py]: Remove path

**Review by mariuscrsn — CHANGES_REQUESTED** [review]

File: code/sys_conf/src/system_config_tool/sys_conf.py (hunk 1 above)
Comment (javibu13) [reply code/sys_conf/src/system_config_tool/sys_conf.py]: Done

File: code/sys_conf/src/system_config_tool/sys_conf.py (hunk 2 above)
Comment (javibu13) [reply code/sys_conf/src/system_config_tool/sys_conf.py]: Done

**Review by mariuscrsn — APPROVED** [review]

**Review by mariuscrsn — APPROVED** [review]

**Pull request merged**